tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -37,4 +37,42 @@ Check Image Build Status
   Run Keyword If    "${build_status}" == "${target_status}"
   ...    Log   The '${build_name}' image match the expected target status
   ...  ELSE
-  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
\ No newline at end of file
+  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
+
+Search Last Build
+    [Documentation]    Returns last build
+    [Arguments]    ${namespace}    ${word-to-search-build}
+    ${build} =    Run    oc get builds -n ${namespace} | grep ${word-to-search-build} | awk '{print $1;}'
+    [Return]    ${build}
+
+Delete Build
+    [Documentation]    Delete the build
+    [Arguments]    ${namespace}    ${build_name}
+    OpenShiftCLI.Delete    kind=Build    namespace=${namespace}    field_selector=metadata.name==${build_name}
+
+Start New Build
+    [Documentation]    Start new Buid
```

Comment:
Same as before (Starts instead of Start)
_Starts new build using ${buildconfig}_